src/core/EdFi.DataManagementService.Core.External/Interface/IApiService.cs

```diff
@@ -2,41 +2,42 @@
 // Licensed to the Ed-Fi Alliance under one or more agreements.
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
+using EdFi.DataManagementService.Core.External.Model;
 using EdFi.DataManagementService.Core.Model;
 
-namespace EdFi.DataManagementService.Core;
+namespace EdFi.DataManagementService.Core.External.Interface;
 
 /// <summary>
-/// The facade a frontend uses to access DMS Core.
+/// The facade a frontend uses to access DMS Core API services.
 ///
 /// The intent of this design is to provide a web framework-independent interface
 /// for 1) ease of testing and 2) ease of supporting future frontends e.g.
 /// AWS Lambda, Azure functions, etc.
 /// </summary>
-public interface ICoreFacade
+public interface IApiService
```

Comment:
Renamed ICoreFacade to IApiService, as "CoreFacade" was too generic a name for the Core.External library.